hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneClientAdapterImpl.java

```diff
@@ -357,6 +357,10 @@ public boolean deleteObject(String keyName, boolean recursive)
       bucket.deleteDirectory(keyName, recursive);
       return true;
     } catch (OMException ome) {
+      if (OMException.ResultCodes.KEY_NOT_FOUND == ome.getResult()) {
+        LOG.warn("delete key failed {}", ome.getMessage());
```

Comment:
it would be really helpful if it can log the key name. The message returned by OM doesn't include the key name.

Reply:
@jojochuang Thanks for review, This case already OM provides key name in exception, eg:

`ERROR ozone.BasicOzoneClientAdapterImpl: delete key failed
Unable to get file status: volume: ozonestage bucket: sparktest key: tmp/tpcds_output/q20.out/.spark-staging-836b5b33-3941-4734-8f90-23abdfee9167`

Reply:
Ok sorry my bad. I looked at OM code and thought it's not included.